Tools/LyTestTools/tests/unit/test_o3de_multi_test_framework.py

```diff
@@ -1017,6 +1018,119 @@ def test_GetNumberParallelEditors_ConfigNotExists_ReturnsDefault(self):
         number_of_executables = mock_test_suite._get_number_parallel_executables(mock_request)
         assert number_of_executables == mock_test_suite.get_number_parallel_executables()
 
+    @mock.patch('ly_test_tools.o3de.editor_test_utils.kill_all_ly_processes')
+    @mock.patch('ly_test_tools.o3de.editor_test_utils.prepare_asset_processor')
+    @mock.patch('ly_test_tools.launchers.launcher_helper.create_editor')
+    @mock.patch('ly_test_tools.launchers.launcher_helper.create_atom_tools_launcher')
+    def test_SetupTest_AtomExe_SetsExecutable(self, mock_create_atom, mock_create_editor, mock_prepare_ap, 
+                                              mock_kill_processes):
+        mock_test_suite = ly_test_tools.o3de.multi_test_framework.MultiTestSuite()
+        mock_test_suite.atom_tools_executable_name = 'mock_atom_exe'
+        mock_test_suite.executable = mock.MagicMock()
+
+        mock_test_suite._setup_test(mock.MagicMock(), mock.MagicMock())
+        assert mock_create_atom.called
```

Comment:
is there an assert_called_with that can validate the mock_create_atom is called with the 'mock_atom_exe' argument?

Reply:
added